lpc82x-hal/Cargo.toml

```diff
@@ -0,0 +1,70 @@
+[package]
+name    = "lpc82x-hal"
+version = "0.4.0"
+authors = ["Hanno Braun <hanno@braun-robotics.com>"]
+edition = "2018"
+
+description   = "Hardware Abstraction Layer (HAL) for the NXP LPC82x series of ARM Cortex-M0+ microcontroller"
+documentation = "https://docs.rs/lpc82x-hal"
+repository    = "https://github.com/braun-robotics/rust-lpc82x-hal"
+license       = "0BSD"
+readme        = "README.md"
+categories    = ["embedded", "hardware-support", "no-std"]
+keywords      = ["arm", "cortex-m", "nxp", "lpc", "embedded-hal-impl"]
+
+
+[badges]
+maintenance = { status = "actively-developed" }
+travis-ci   = { repository = "braun-robotics/rust-lpc82x-hal" }
+
+
+[dependencies]
+cortex-m     = "0.5.2"
+embedded-hal = { version = "0.2", features = ["unproven"] }
+lpc82x-pac   = "0.6"
+nb           = "0.1.1"
+void         = { version = "1", default-features = false }
+# This should be in [dev-dependencies], but those can't be optional. Issue:
+# https://github.com/rust-lang/cargo/issues/1596
+compiletest_rs = { version = "0.3", optional = true }
+
+[dependencies.lpc8xx-hal-common]
+path = "../lpc8xx-hal-common"
+features = ["82x"]
```

Comment:
Same comment applies that I left in the other `Cargo.toml`. I believe we need to combine both `path` and `version` here, otherwise we can't publish it on crates.io.